src/renderer/components/Cards/NodeCard.tsx

```diff
@@ -1,20 +1,35 @@
-import * as React from "react";
+import React from "react";
+import ReactTooltip from "react-tooltip";
 
 export interface INodeCardProps {
     title: string;
     value: number | string;
     url: string;
     onClick: () => void;
+    isSyncing?: boolean;
```

Comment:
Can value be -1 if not syncing to avoid this additional flag in interface?

Reply:
Yep!

Reply:
Actually, I am not sure how API works. What if it stops syncing but stops at some height? Currently we display either it is syncing and what's the current height. Otherwise, we can't tell if finished syncing or not.

Reply:
Well, you'll subscribe to new head to update slot number and you can check syncing status every time (or every other)